### PR TITLE
fixes #8

### LINKED ISSUE #8: Please check in template secrets.yml
Want to get it running on another PC but don't want to reverse engineer the yaml again. Thx :)

### PR DESCRIPTION
**Pull request by mrjkey**

### CODE REVIEW

File: example_secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file needs to be in a directory called "ignore"
```
Comment (mrjkey) [inline example_secrets.yaml:1]:
```suggestion
# This file needs to be in a directory called "ignore". Rename it to 'secrets.yaml' and provide the correct keys

**Review by Tris20 — CHANGES_REQUESTED** [review]
Hmmm, can we have slightly better example keys in a public repo please? :)

**Comment by mrjkey** [comment]
> Hmmm, can we have slightly better example keys in a public repo please? :)

Absolutely, please use the "```suggestion" feature in github to add those

File: example_secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This file needs to be in a directory called "ignore". Rename it to 'secrets.yaml' and provide the correct keys
+
+api_keys:
+  openweathermap.org: "dick &"
+  partybus: "balls"
```
Comment (Tris20) [inline example_secrets.yaml:5]:
```suggestion
  openweathermap.org: "abc123def456ghi789jkl012mno345pqr678stu90"
  partybus: "abc123def456ghi789jkl012mno345pqr678stu90" 
```

**Review by Tris20 — CHANGES_REQUESTED** [review]

**Pull request merged**